### PR TITLE
CDAP-1908: Moved cdap-security-config.json and cdap-config.json to a better place

### PR DESCRIPTION
**Pull request by jwang47**
https://issues.cask.co/browse/CDAP-1908
http://builds.cask.co/browse/CDAP-RBT211

In Standalone run from IDE, generated config files for the UI go to `cdap-web-app/conf/generated` now. In Standalone run from SDK, generated config files should go to `<sdk-base-dir>/conf/generated` (still need to test it)

### CODE REVIEW

File: cdap-standalone/src/main/java/co/cask/cdap/WebCloudAppService.java [hunk 1]
```diff
@@ -90,8 +91,10 @@ public WebCloudAppService(@Named("web-app-path")String webAppPath, CConfiguratio
    */
   @Override
   protected void startUp() throws Exception {
-    generateConfigFile(new File(webAppBase, JSON_PATH), cConf);
-    generateConfigFile(new File(webAppBase, JSON_SECURITY_PATH), sConf);
+    File confDir = new File(new File(webAppBase, "conf"), "generated");
+    Preconditions.checkState(confDir.mkdirs(), "Couldn't create directory for generated conf files for the UI");
```
Comment (nitinmotgi) [inline cdap-standalone/src/main/java/co/cask/cdap/WebCloudAppService.java]: Will this fail if directory already exists ?

**Comment by nitinmotgi** [comment]
minor comment. If that is not a problem then - lgtm :+1:

**Comment by nitinmotgi** [comment]
lgtm :+1:

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**